simpleviewinc/keg-cli
# Remove regulator

**Ticket**: https://jira.simpleviewtools.com/browse/ZEN-346

## Context

* Keg regulator will be its own Tap/App and should not be configured inside keg-cli
* `keg-regulator` repo should be linked as an external Tap/App
* remove the keg-regulator `containers` and `tasks` and any other references to keg-container directly
* I ran the unit test on master and it seems like 1 test is failing, so I updated that as well

## Goal

* Remove direct references and configuration to `keg-regulator`

## Updates
* `src/utils/getters/getContext.js`
   * updated because the initial unit test failed. let me know if this is not the right update

* `containers/regulator/*`
    * removed all files

* `scripts/setup/cli.config.json`
   * removed keg regulator ref

* `scripts/setup/defaults.env`
   * removed keg regulator ref

* `src/constants/constants.js`
   * removed keg regulator ref

* `src/tasks/regulator/*`
    * removed all files. this should be inside `keg-regulator` repo

* `src/utils/builders/__tests__/buildCmdContext.js`
   * updated keg-reg test reference

* `/Users/dyo/keg/keg-cli/src/libs/fileSys/__tests__/*`
- updated the name of temp yml file
- it fails sometimes when you run the whole unit tests because the test from yml.js/parseTemplate.js modifies/deletes the same file

* updated yarn test to run with no cache

## Testing

* pull branch
* run `yarn test` - ensure all test passes
* run `keg regulator` - should be an invalid command (unless you have a link tap called regulator)
